Deeploy/Targets/NE16/Deployer.py

```
@@ -0,0 +1,40 @@
+class NE16Deployer(GAP9Deployer):
+
+    def __init__(self,
+                 graph: gs.Graph,
+                 deploymentPlatform: DeploymentPlatform,
+                 inputTypes: Dict[str, Type[Pointer]],
+                 loweringOptimizer: TopologyOptimizer,
+                 scheduler: Callable = lambda graph: list(graph.nodes),
+                 name: str = 'DeeployNetwork',
+                 default_channels_first = False,
+                 deeployStateDir: str = "DeeployStateDir",
+                 inputOffsets = {}):
```

Ruff (0.15.9)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)